### ISSUE TITLE
vagrant-hostmanager missing, please install the plugin with this command

### ISSUE THREAD

**Issue by connerbw** [hashicorp/vagrant#10153]
On Windows 10, using choco, I upgraded Vagrant. It stopped working. I now get:

> Vagrant failed to initialize at a very early stage:
> vagrant-hostmanager missing, please install the plugin with this command

The plugin **_is_** installed. 

I ran `rm -Rf ~/.vagrant.d/gems/ && rm ~/.vagrant.d/plugins.json` reinstalled the plugin using `vagrant plugin install vagrant-hostmanage` and the error persists.

```
me@COMPY C:\Users\me\Code\pressbooks-dev\trellis
# vagrant -v
Vagrant 2.1.3

me@COMPY C:\Users\me\Code\pressbooks-dev\trellis
# vagrant up
Vagrant failed to initialize at a very early stage:

vagrant-hostmanager missing, please install the plugin with this command:
vagrant plugin install vagrant-hostmanager

Or install landrush for multisite subdomains:
vagrant plugin install landrush

me@COMPY C:\Users\me\Code\pressbooks-dev\trellis
# vagrant plugin list
vagrant-bindfs (1.1.0, global)
vagrant-hostmanager (1.8.9, global)
```

**Comment by connerbw** [comment]
Possibly related: https://github.com/devopsgroup-io/vagrant-hostmanager/issues/225

**Comment by briancain** [comment]
Have you tried uninstalling Vagrant, making sure its files are gone in the install directory, and reinstalling it? Also having a debug gist when you run `vagrant up --debug` would make this easier to see what is going on. Thanks!

**Comment by y0y0z** [comment]
possibly related to https://github.com/hashicorp/vagrant/issues/10150

**Comment by connerbw** [comment]
> Have you tried uninstalling Vagrant, making sure its files are gone in the install directory, and reinstalling it? 

Ok, done.

I also installed  the `vagrant-winnfsd` plugin between now and my last post because I use it and it was missing, thinking it might help.

```
Microsoft Windows [Version 10.0.17134.228]

me@COMPY C:\Users\me\Code\pressbooks-dev\trellis
$ vagrant -v
Vagrant 2.1.3

me@COMPY C:\Users\me\Code\pressbooks-dev\trellis
$ vagrant plugin list
vagrant-bindfs (1.1.0, global)
vagrant-hostmanager (1.8.9, global)
vagrant-winnfsd (1.4.0, global)

me@COMPY C:\Users\me\Code\pressbooks-dev\trellis
$ vagrant up --debug
 INFO global: Vagrant version: 2.1.3
 INFO global: Ruby version: 2.4.4
 INFO global: RubyGems version: 2.6.14.1
 INFO global: VAGRANT_EXECUTABLE="C:\\HashiCorp\\Vagrant\\embedded\\gems\\2.1.3\\gems\\vagrant-2.1.3\\bin\\vagrant"
 INFO global: VAGRANT_INSTALLER_EMBEDDED_DIR="C:\\HashiCorp\\Vagrant\\embedded"
 INFO global: VAGRANT_INSTALLER_ENV="1"
 INFO global: VAGRANT_INSTALLER_VERSION="2"
 INFO global: VAGRANT_LOG="debug"
 WARN global: resolv replacement has not been enabled!
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/box/plugin.rb
 INFO manager: Registered plugin: box command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/cap/plugin.rb
 INFO manager: Registered plugin: cap command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/destroy/plugin.rb
 INFO manager: Registered plugin: destroy command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/global-status/plugin.rb
 INFO manager: Registered plugin: global-status command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/halt/plugin.rb
 INFO manager: Registered plugin: halt command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/help/plugin.rb
 INFO manager: Registered plugin: help command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/init/plugin.rb
 INFO manager: Registered plugin: init command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/list-commands/plugin.rb
 INFO manager: Registered plugin: list-commands command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/login/plugin.rb
 INFO manager: Registered plugin: vagrant-login
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/package/plugin.rb
 INFO manager: Registered plugin: package command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/plugin/plugin.rb
 INFO manager: Registered plugin: plugin command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/port/plugin.rb
 INFO manager: Registered plugin: port command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/powershell/plugin.rb
 INFO manager: Registered plugin: powershell command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/provider/plugin.rb
 INFO manager: Registered plugin: provider command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/provision/plugin.rb
 INFO manager: Registered plugin: provision command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/push/plugin.rb
 INFO manager: Registered plugin: push command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/rdp/plugin.rb
 INFO manager: Registered plugin: rdp command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/reload/plugin.rb
 INFO manager: Registered plugin: reload command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/resume/plugin.rb
 INFO manager: Registered plugin: resume command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/snapshot/plugin.rb
 INFO manager: Registered plugin: snapshot command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/ssh/plugin.rb
 INFO manager: Registered plugin: ssh command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/ssh_config/plugin.rb
 INFO manager: Registered plugin: ssh-config command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/status/plugin.rb
 INFO manager: Registered plugin: status command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/suspend/plugin.rb
 INFO manager: Registered plugin: suspend command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/up/plugin.rb
 INFO manager: Registered plugin: up command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/validate/plugin.rb
 INFO manager: Registered plugin: validate command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/commands/version/plugin.rb
 INFO manager: Registered plugin: version command
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/communicators/ssh/plugin.rb
 INFO manager: Registered plugin: ssh communicator
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/communicators/winrm/plugin.rb
 INFO manager: Registered plugin: winrm communicator
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/communicators/winssh/plugin.rb
 INFO manager: Registered plugin: windows ssh communicator
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/alt/plugin.rb
 INFO manager: Registered plugin: ALT Platform guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/amazon/plugin.rb
 INFO manager: Registered plugin: Amazon Linux guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/arch/plugin.rb
 INFO manager: Registered plugin: Arch guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/atomic/plugin.rb
 INFO manager: Registered plugin: Atomic Host guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/bsd/plugin.rb
 INFO manager: Registered plugin: BSD-based guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/coreos/plugin.rb
 INFO manager: Registered plugin: CoreOS guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/darwin/plugin.rb
 INFO manager: Registered plugin: Darwin guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/debian/plugin.rb
 INFO manager: Registered plugin: Debian guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/dragonflybsd/plugin.rb
 INFO manager: Registered plugin: DragonFly BSD guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/elementary/plugin.rb
 INFO manager: Registered plugin: Elementary guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/esxi/plugin.rb
 INFO manager: Registered plugin: ESXi guest.
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/fedora/plugin.rb
 INFO manager: Registered plugin: Fedora guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/freebsd/plugin.rb
 INFO manager: Registered plugin: FreeBSD guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/funtoo/plugin.rb
 INFO manager: Registered plugin: Funtoo guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/gentoo/plugin.rb
 INFO manager: Registered plugin: Gentoo guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/haiku/plugin.rb
 INFO manager: Registered plugin: Haiku guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/kali/plugin.rb
 INFO manager: Registered plugin: Kali guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/linux/plugin.rb
 INFO manager: Registered plugin: Linux guest.
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/mint/plugin.rb
 INFO manager: Registered plugin: Mint guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/netbsd/plugin.rb
 INFO manager: Registered plugin: NetBSD guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/nixos/plugin.rb
 INFO manager: Registered plugin: NixOS guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/omnios/plugin.rb
 INFO manager: Registered plugin: OmniOS guest.
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/openbsd/plugin.rb
 INFO manager: Registered plugin: OpenBSD guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/photon/plugin.rb
 INFO manager: Registered plugin: VMware Photon guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/pld/plugin.rb
 INFO manager: Registered plugin: PLD Linux guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/redhat/plugin.rb
 INFO manager: Registered plugin: Red Hat Enterprise Linux guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/slackware/plugin.rb
 INFO manager: Registered plugin: Slackware guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/smartos/plugin.rb
 INFO manager: Registered plugin: SmartOS guest.
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/solaris/plugin.rb
 INFO manager: Registered plugin: Solaris guest.
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/solaris11/plugin.rb
 INFO manager: Registered plugin: Solaris 11 guest.
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/suse/plugin.rb
 INFO manager: Registered plugin: SUSE guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/tinycore/plugin.rb
 INFO manager: Registered plugin: TinyCore Linux guest.
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/trisquel/plugin.rb
 INFO manager: Registered plugin: Trisquel guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/ubuntu/plugin.rb
 INFO manager: Registered plugin: Ubuntu guest
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/guests/windows/plugin.rb
 INFO manager: Registered plugin: Windows guest.
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/alt/plugin.rb
 INFO manager: Registered plugin: ALT Platform host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/arch/plugin.rb
 INFO manager: Registered plugin: Arch host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/bsd/plugin.rb
 INFO manager: Registered plugin: BSD host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/darwin/plugin.rb
 INFO manager: Registered plugin: Mac OS X host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/freebsd/plugin.rb
 INFO manager: Registered plugin: FreeBSD host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/gentoo/plugin.rb
 INFO manager: Registered plugin: Gentoo host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/linux/plugin.rb
 INFO manager: Registered plugin: Linux host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/null/plugin.rb
 INFO manager: Registered plugin: null host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/redhat/plugin.rb
 INFO manager: Registered plugin: Red Hat Enterprise Linux host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/slackware/plugin.rb
 INFO manager: Registered plugin: Slackware host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/suse/plugin.rb
 INFO manager: Registered plugin: SUSE host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/hosts/windows/plugin.rb
 INFO manager: Registered plugin: Windows host
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/kernel_v1/plugin.rb
 INFO manager: Registered plugin: kernel
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/kernel_v2/plugin.rb
 INFO manager: Registered plugin: kernel
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/providers/docker/plugin.rb
 INFO manager: Registered plugin: docker-provider
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/providers/hyperv/plugin.rb
 INFO manager: Registered plugin: Hyper-V provider
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/providers/virtualbox/plugin.rb
 INFO manager: Registered plugin: VirtualBox provider
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/provisioners/ansible/plugin.rb
 INFO manager: Registered plugin: ansible
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/provisioners/cfengine/plugin.rb
 INFO manager: Registered plugin: CFEngine Provisioner
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/provisioners/chef/plugin.rb
 INFO manager: Registered plugin: chef
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/provisioners/docker/plugin.rb
 INFO manager: Registered plugin: docker
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/provisioners/file/plugin.rb
 INFO manager: Registered plugin: file
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/provisioners/puppet/plugin.rb
 INFO manager: Registered plugin: puppet
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/provisioners/salt/plugin.rb
 INFO manager: Registered plugin: salt
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/provisioners/shell/plugin.rb
 INFO manager: Registered plugin: shell
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/pushes/atlas/plugin.rb
 INFO manager: Registered plugin: atlas
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/pushes/ftp/plugin.rb
 INFO manager: Registered plugin: ftp
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/pushes/heroku/plugin.rb
 INFO manager: Registered plugin: heroku
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/pushes/local-exec/plugin.rb
 INFO manager: Registered plugin: local-exec
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/pushes/noop/plugin.rb
 INFO manager: Registered plugin: noop
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/synced_folders/nfs/plugin.rb
 INFO manager: Registered plugin: NFS synced folders
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/synced_folders/rsync/plugin.rb
 INFO manager: Registered plugin: RSync synced folders
DEBUG global: Loading core plugin: C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/plugins/synced_folders/smb/plugin.rb
 INFO manager: Registered plugin: SMB synced folders
 INFO vagrant: `vagrant` invoked: ["up", "--debug"]
DEBUG vagrant: Creating Vagrant environment
 INFO environment: Environment initialized (#<Vagrant::Environment:0x0000000004da71f0>)
 INFO environment:   - cwd: C:/Users/dac/Code/pressbooks-dev/trellis
 INFO environment: Home path: C:/Users/dac/.vagrant.d
DEBUG environment: Effective local data path: C:/Users/dac/Code/pressbooks-dev/trellis/.vagrant
 INFO environment: Local data path: C:/Users/dac/Code/pressbooks-dev/trellis/.vagrant
DEBUG environment: Creating: C:/Users/dac/Code/pressbooks-dev/trellis/.vagrant
DEBUG manager: Enabling localized plugins
 INFO manager: Plugins:
DEBUG bundler: Current generated plugin dependency list: []
DEBUG bundler: Generating new builtin set instance.
DEBUG bundler: Generating new plugin set instance. Skip gems - []
DEBUG bundler: Activating solution set: []
 INFO loader: Set :root = ["#<Pathname:C:/Users/dac/Code/pressbooks-dev/trellis/Vagrantfile>"]
DEBUG loader: Populating proc cache for #<Pathname:C:/Users/dac/Code/pressbooks-dev/trellis/Vagrantfile>
DEBUG loader: Load procs for pathname: C:/Users/dac/Code/pressbooks-dev/trellis/Vagrantfile
 INFO root: Version requirements from Vagrantfile: [">= 2.0.1"]
 INFO root:   - Version requirements satisfied!
 INFO loader: Loading configuration in order: [:home, :root]
DEBUG loader: Loading from: root (evaluating)
ERROR vagrant: Vagrant experienced an error! Details:
ERROR vagrant: #<Vagrant::Errors::VagrantError: vagrant-hostmanager missing, please install the plugin with this command:
vagrant plugin install vagrant-hostmanager

Or install landrush for multisite subdomains:
vagrant plugin install landrush>
ERROR vagrant: vagrant-hostmanager missing, please install the plugin with this command:
vagrant plugin install vagrant-hostmanager

Or install landrush for multisite subdomains:
vagrant plugin install landrush
ERROR vagrant: C:/Users/dac/Code/pressbooks-dev/trellis/lib/trellis/vagrant.rb:37:in `fail_with_message'
C:/Users/dac/Code/pressbooks-dev/trellis/Vagrantfile:67:in `block in <top (required)>'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/config/v2/loader.rb:37:in `load'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/config/loader.rb:126:in `block (2 levels) in load'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/config/loader.rb:119:in `each'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/config/loader.rb:119:in `block in load'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/config/loader.rb:116:in `each'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/config/loader.rb:116:in `load'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/vagrantfile.rb:28:in `initialize'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/environment.rb:792:in `new'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/environment.rb:792:in `vagrantfile'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/lib/vagrant/environment.rb:171:in `initialize'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/bin/vagrant:144:in `new'
C:/HashiCorp/Vagrant/embedded/gems/2.1.3/gems/vagrant-2.1.3/bin/vagrant:144:in `<main>'
Vagrant failed to initialize at a very early stage:

vagrant-hostmanager missing, please install the plugin with this command:
vagrant plugin install vagrant-hostmanager

Or install landrush for multisite subdomains:
vagrant plugin install landrush
```

**Comment by connerbw** [comment]
I hope this is is enough? I'm going to attempt a downgrade to 2.1.2 ( as seen in https://github.com/hashicorp/vagrant/issues/10153#issuecomment-417371869 ) because I have to get some work done.

**Comment by briancain** [comment]
Hi @connerbw - We are currently looking at patching the related plugin issues linked here on the latest release of 2.1.3. For now I recommend downgrading to the previous version. Thanks! :)

**Comment by connerbw** [comment]
I downgraded to 2.1.2 but I am still experiencing some weirdness.

running e.g. `vagrant plugin list` outside the trellis folder returns the correct list of plugins.

However within the `trellis` [1] folder I can’t run the same command. It returns vagrant-hostmanager missing error.

At least vagrant up works and I can do some dev work (for now.)

[1]  https://github.com/roots/trellis

**Comment by robrecord** [comment]
Was this patch released on 2.1.4? The issue persists.
